src/integrator_interface.jl

```diff
@@ -400,6 +400,7 @@ function Base.iterate(integrator::DEIntegrator,state=0)
 end
 
 Base.eltype(integrator::DEIntegrator) = typeof(integrator)
+Base.IteratorSize(::Type{T}) where T<:DEIntegrator = Base.SizeUnknown()
```

Comment:
It seems the type parameter is not needed:
```suggestion
Base.IteratorSize(::Type{<:DEIntegrator}) = Base.SizeUnknown()
```